unison-cli/src/Unison/Cli/Monad.hs

```diff
@@ -246,6 +253,14 @@ data LoadSourceResult
   | LoadError
   | LoadSuccess Text
 
+-- | The result of calling 'loadSource'.
+data WriteSourceAction
+  = -- Append the given text to the source. (include fold, contents)
+    PrependSource Bool Text
+  | -- Overwrite the source with the given text.
+    OverwriteSource Text
```

Comment:
I think it may be premature to add this parameter, because right now the transcript runner would _only_ be overwriting (for transcript brevity), and ucm would _only_ be prepending.

I could imagine needing something like this if we get into a situation where ucm would do something other than prepending, but I thing we will basically be prepending as long as we are using scratch files in external editors.

Reply:
@ChrisPenner I propose removing this parameter and then merging, but let me know if I missed a reason why it's important to have it now.

Reply:
I added the fold parameter because `> display.to` currently prepends without a fold, but I'm not really sure what that command is used for, 'display' output isn't always valid scratch-file output 🤔

Reply:
We chatted in slack, we're not really sure what `display.to`'s main use-case is, but it's not really rendering parseable source, so I changed it to use a simple file prepend rather than using writeSource.